apps/penxle.com/src/routes/editor/[permalink]/Editor.svelte

```diff
@@ -86,15 +87,21 @@
 
   const makeRevisePost = (revisionKind: PostRevisionKind) => {
     return async () => {
-      const resp = await revisePost({
-        revisionKind,
-        postId: $post.id,
-        title: $store.title,
-        subtitle: $store.subtitle,
-        content: $store.content,
-        paragraphIndent: $store.paragraphIndent,
-        paragraphSpacing: $store.paragraphSpacing,
-      });
+      let resp;
```

Comment:
```suggestion
      let resp: Awaited<ReturnType<typeof revisePost>> | undefined;
      try {
        resp = await revisePost({
      }
      // 중략
      if (!resp) throw new Error('No response');
```

P2: `resp` 타입 명시 부탁드립니다!